client/web/src/enterprise/campaigns/detail/changesets/DownloadDiffButton.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useState } from 'react'
+import FileDownloadIcon from 'mdi-react/FileDownloadIcon'
+import { LoadingSpinner } from '@sourcegraph/react-loading-spinner'
+import { getChangesetDiff } from '../backend'
+import { asError } from '../../../../../../shared/src/util/errors'
+import AlertCircleIcon from 'mdi-react/AlertCircleIcon'
+
+export interface DownloadDiffButtonProps {
+    changesetID: string
+}
+
+enum DownloadState {
+    Ready,
+    Loading,
+    Error,
```

Comment:
TS enums are often upper-case, but not a blocker :)
```suggestion
enum DownloadState {
    READY,
    LOADING,
    ERROR,
```

Reply:
Thanks. Sorry, Rust showing through a bit there!

Reply:
I actually identified as a go-ism from what Kubernetes usually does for enum values 😬